packages/wallet/src/pages/index.tsx

```diff
@@ -1,48 +1,82 @@
-      <PageHeader
-        title="Hello, John Doe!"
-        message="Here is your account overview!"
-      />
-      <div className="mt-10 flex flex-col space-y-10">
-        <div>
+      <div className="flex items-center justify-between md:flex-col md:items-start md:justify-start">
+        <PageHeader
+          title="Hello, John Doe!"
+          message="Here is your account overview!"
+        />
+        <div className="text-green md:mt-10">
           <h2 className="text-lg font-light md:text-xl">Balance</h2>
           <p className="text-2xl font-semibold md:text-4xl">$2,934</p>
         </div>
-        <div className="flex w-full flex-col space-y-2 md:max-w-lg">
-          <div className="flex items-center justify-between">
-            <h3 className="text-lg font-semibold">My Accounts</h3>
-            <Button intent="outline" size="sm" aria-label="manage accounts">
-              Manage Accounts
-            </Button>
-          </div>
-          <div className="flex flex-col space-y-5">
-            {accounts.map((account) => (
-              <Link
-                href={`accounts/${account.id}`}
-                key={account.id}
-                className="flex items-center justify-between rounded-md bg-turqoise px-4 py-4"
-              >
-                <span className="font-semibold">{account.name}</span>
-                <span>
-                  {account.balance} {account.asset.code}
-                </span>
-              </Link>
-            ))}
-          </div>
+      </div>
+      <div className="flex w-full flex-col space-y-3 md:max-w-md">
+        <div className="mt-2 grid grid-cols-4 gap-2">
+          <Link className="group flex aspect-square flex-col items-center justify-center rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6">
+            <Pay className="h-8 w-8" />
+            <span className="font-medium text-green-5 group-hover:text-green-6">
+              Send
+            </span>
+          </Link>
+          <Link className="group flex aspect-square flex-col items-center justify-center rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6">
+            <Send className="h-8 w-8" />
+            <span className="font-medium text-green-5 group-hover:text-green-6">
+              Pay
+            </span>
+          </Link>
+          <Link className="group flex aspect-square flex-col items-center justify-center rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6">
+            <Receive className="h-8 w-8" />
+            <span className="font-medium text-green-5 group-hover:text-green-6">
+              Receive
+            </span>
+          </Link>
+          <Link className="group flex aspect-square flex-col items-center justify-center rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6">
+            <New className="h-8 w-8" />
+            <span className="font-medium text-green-5 group-hover:text-green-6">
```

Comment:
```suggestion
            <span className="font-medium leading-3 text-green-5 group-hover:text-green-6 sm:leading-none">
```

Reply:
By doing this, the icons would not be correctly aligned on mobile devices. 🤔